pkg/template/jsonpath_test.go

```diff
@@ -213,3 +218,48 @@ func TestRelaxedJSONPathExpression_Error(t *testing.T) {
 		})
 	}
 }
+
+func TestFindTektonExpressions(t *testing.T) {
+	tcs := []struct {
+		in   string
+		want []string
+	}{{
+		in:   "$(body.blah)",
```

Comment:
Let's add a `""` case for sanity checking.

Reply:
done!